examples/node/scripts/createDAPostViaDispatcher.ts

```diff
@@ -0,0 +1,67 @@
+import { isCreateDataAvailabilityPublicationResult } from "@lens-protocol/client";
+import { getAuthenticatedClient } from "./shared/getAuthenticatedClient";
+import { setupWallet } from "./shared/setupWallet";
+import { uploadWithBundlr } from "./shared/uploadWithBundlr";
+import { getActiveProfile } from "./shared/getActiveProfile";
+import { buildPublicationMetadata } from "./shared/buildPublicationMetadata";
+
+async function main() {
+  const wallet = setupWallet();
+  const address = await wallet.getAddress();
+  const lensClient = await getAuthenticatedClient(wallet);
+  const profile = await getActiveProfile(lensClient, address);
+  const profileId = profile.id;
+
+  // prepare metadata
+  const metadata = buildPublicationMetadata({
+    content: "Data Availability Post created with LensClient SDK",
+    name: "Data Availability Post created with LensClient SDK",
+  });
+
+  // validate metadata
+  const validateResult = await lensClient.publication.validateMetadata(metadata);
+
+  if (!validateResult.valid) {
+    throw new Error(`Metadata is not valid.`);
+  }
+
+  // upload metadata to
+  const contentURI = await uploadWithBundlr(metadata);
+
+  console.log(`Post metadata was uploaded to ${contentURI}`);
+
+  // create a post via dispatcher, you need to have the dispatcher enabled for the profile
+  const createPostResult = await lensClient.publication.createDataAvailabilityPostViaDispatcher({
+    from: profileId,
+    contentURI,
+  });
+
+  // createPostResult is a Result object
+  const createPostResultValue = createPostResult.unwrap();
+
+  if (!isCreateDataAvailabilityPublicationResult(createPostResultValue)) {
+    console.log(`Something went wrong`, createPostResultValue);
+    return;
+  }
+
+  console.log(`DA post was created: `, createPostResultValue);
+
+  // DA post is created instantly, so we can also comment on it right away
+  const createCommentResult =
+    await lensClient.publication.createDataAvailabilityCommentViaDispatcher({
+      from: profileId,
+      contentURI,
+      commentOn: createPostResultValue.id,
+    });
+
+  const createCommentResultValue = createCommentResult.unwrap();
+
+  if (!isCreateDataAvailabilityPublicationResult(createCommentResultValue)) {
+    console.log(`Something went wrong`, createCommentResultValue);
```

Comment:
Do we need to consider this as template strings?

or we can make use of it

```suggestion
    console.log(`Something went wrong ${createCommentResultValue}`);
```

Reply:
no, it won't work, as I want to print out an object, not a string, so with your suggestion it ends up as `[object Object]`.  
Also please don't block a PR with "request changes" if you only want to suggest something insignificant in the scope of the PR.

compare
```
console.log(`Transaction status: `, wasIndexedFirstCheck.unwrap());
// vs.
console.log(`Transaction status: ${wasIndexedFirstCheck.unwrap()}`);
```

result
```
Transaction status:  {
  __typename: 'TransactionIndexedResult',
  indexed: false,
  txHash: '0xdfbf48b549813fc1c4654122132223d4ceccc9b1b6e71dbbec609560b4838fc0'
}
// vs.
Transaction status: [object Object]
```